server/face_recognition/test.py

```diff
@@ -0,0 +1,19 @@
+from compreface import CompreFace
+from compreface.service import RecognitionService
+from compreface.collections import FaceCollection
+from compreface.collections.face_collections import Subjects
+
+# https://github.com/exadel-inc/compreface-python-sdk
+
+DOMAIN: str = 'http://100.106.18.99'
```

Comment:
Same as other PR, maybe just have 1 file with all constants and import these strings here